Update user access management docs (#622)
astronomer/docs: astro/manage-workspaces.md

## Manage Workspace Users

> As a Workspace Admin, you can add or remove users from your Workspace at any time. Before adding a user, make sure the user is an existing member of the Organization or [invite](add-user.md#step-1-invite-the-user-to-astro) them. To add a user to a Workspace:

Reviewer: ```suggestion
As a Workspace Admin, you can add or remove users from your Workspace at any time. Before adding a user, confirm that the user is an existing member of the Organization or [invite](add-user.md#step-1-invite-the-user-to-astro) them. To add a user to a Workspace:
```

1. In the Workspace view, go to the **Access** page.
2. Click **Add member**.
3. Specify the user's email and role within the Workspace. For more information about Workspace roles and permissions, see [User Permissions](user-permissions.md).

> Once added, the user will see the Workspace listed on the **Overview** page. They will also appear as an active Workspace member in the **Access** tab. For more information about inviting users to Astronomer, see [Add User](add-user.md).

Reviewer: ```suggestion
Once added, the user will see the Workspace listed on the **Overview** page of the Cloud UI. They will also appear as an active Workspace member in the **Access** tab. For more information about inviting users to Astronomer, see [Add User](add-user.md).
```

## Manage Workspace Users

> :::info
> 
> The user must be an existing member of the Organization, to add them as a user of a Workspace.
> 
> :::
> 

Reviewer: ```suggestion
```

@chomipi88 Instead of this info at the top I think I'd either just:
- Add this note as "caution" below if we feel like it's less common to run into this
- Add a sentence before the 3 steps here that says that "first, make sure the user is listed as an existing member of the Organization" if we feel like we need to emphasize this requirement

Author: Thought's on the latest commit?

> As a Workspace Admin, you can add or remove users from your Workspace. To add a user to a Workspace:

Reviewer: ```suggestion
As a Workspace Admin, you can add or remove users from your Workspace at any time. To add a user to a Workspace:
```

1. In the Workspace view, go to the **Access** page.
2. Click **Add member**.
3. Specify the user's email and role within the Workspace. For more information about Workspace roles and permissions, see [User Permissions](user-permissions.md).

Once added, the user will see the Workspace listed on the **Overview** page. They will also appear as an active Workspace member in the **Access** tab. For more information about inviting users to Astronomer, see [Add User](add-user.md).

### Update or Remove a Workspace User

To update a Workspace user's access to the Workspace:

1. In the Workspace view, go to the **Access** page.
2. Click the **Edit** button next to the user's entry in the **Member** table:

    ![Edit Workspace user button](/img/docs/edit-workspace-user.png)

> From here, you can update a user's Workspace role, or remove the user from the Workspace.

Reviewer: ```suggestion
From here, you can either update a user's Workspace role or remove the user from the Workspace.
```
